src/suffer.cpp

```diff
@@ -737,11 +737,9 @@ void Character::suffer_in_sunlight()
         vitamin_mod( vitamin_id( "vitC" ), 1, true );
     }
 
-    if( x_in_y( sunlight_nutrition, 12000 ) ) {
-        mod_hunger( -1 );
-        // photosynthesis absorbs kcal directly
-        mod_stored_nutr( -1 );
-        stomach.ate();
+    if( x_in_y( sunlight_nutrition, 6000 ) ) {
```

Comment:
```suggestion
    if( x_in_y( sunlight_nutrition, to_seconds<int>( 100_minutes ) ) ) {
```

Reply:
I'll see if it compiles and commit it if it works.